src/coreclr/tests/scripts/run-corefx-tests.py

```diff
@@ -297,7 +297,7 @@ def main(args):
 
     # Gather up some arguments to pass to the different build scripts.
 
-    common_config_args = '-configuration Release -framework net5.0 -os %s -arch %s' % (clr_os, arch)
+    common_config_args = '-configuration Release -framework net6.0 -os %s -arch %s' % (clr_os, arch)
```

Comment:
Who uses that script? cc @trylek

Reply:
No idea really, I suspect this is some old pre-consolidation junk.